molgenis_documentation/technical_documentation.md

```diff
@@ -22,88 +22,36 @@ The “hairball” is the twisted, tangled maze of teaching that is going to tak
 * Contribution policies (if your project accepts outside contributions).
 
 
-# MOLGENIS Technical documentation
+# MOLGENIS user documentation
 ---
+## Introduction
```

Comment:
How will we let them get their own molgenis? Cause it's big bad overkill to let them compile it from scratch if they can get the latest war from maven central or try it out on a demo server.